mimesis/utils.py

```diff
@@ -114,6 +115,13 @@ def download_image(url: str = '', save_path: str = '',
 
     if url:
         image_name = url.rsplit('/')[-1]
+
+        # check if image name has two parts: name and ext to save it properly
```

Comment:
I guess that it's better to always generate a random name for all images.

Reply:
@conformist-mw What do you think? Can we remove comment and add random names for every downloaded image?